CredentialProvider.Microsoft/CredentialProviders/Vsts/VstsCredentialProvider.cs

```diff
@@ -115,7 +115,7 @@ public override async Task<GetAuthenticationCredentialsResponse> HandleRequestAs
 
                     if (!string.IsNullOrWhiteSpace(sessionToken))
                     {
-                        Verbose(string.Format(Resources.VSTSSessionTokenCreated, request.Uri.ToString()));
+                        Verbose(string.Format(Resources.VSTSSessionTokenCreated, request.Uri.AbsoluteUri.ToString()));
```

Comment:
`AbsoluteUri` is already a `string` (this occurs in several places)

Reply:
done